packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -27,8 +27,10 @@ const ElementRemove = (element: Element): Element =>
     ReflectApply(ElementProtoRemove, element, emptyArray);
 const NodeAppendChild = (parent: Node, child: ChildNode): ChildNode =>
     ReflectApply(NodeProtoAppendChild, parent, [child]);
+// Because it's impossible to test whether NodeLastChild is called below,
+// ignore the function body in coverage reporting.
```

Comment:
If the function is never called can it just be
```js
//    
// istanbul ignore next
```

Reply:
You're right, it can be! This was a bit too surgical 🤣